packages/web/app/src/components/layouts/target.tsx

```diff
@@ -366,144 +366,291 @@ export function ConnectSchemaModal(props: {
     )?.node;
   }, [selectedGraph]);
 
+  const target = query.data?.target;
+
+  const [selectedArtifact, setSelectedArtifact] = useResetState<CdnArtifactType>(
+    () => (target?.project.type === ProjectType.Federation ? 'supergraph' : 'sdl'),
+    [target?.project.type],
+  );
+
   return (
     <Dialog open={props.isOpen} onOpenChange={props.toggleModalOpen}>
-      <DialogContent className="w-[600px] max-w-[700px] gap-5 md:w-3/5">
+      <DialogContent className="w-[650px] min-w-[650px]">
```

Comment:
Modal styles are a bit weird, I set this width to 650px and the body max width to 600px, otherwise content would always go out of bounds, could not find a more reliable way to set content width to be DialogContent's width - the padding.